src/components/ui/events/MainTable.js

```diff
@@ -32,6 +32,14 @@ mainTable.getEventInfo = function(eventRowNum){
     });
 };
 
+mainTable.checkAllMessagesToContain = function(text) {
```

Comment:
I have defined a class for column in the events main table - message should be just another instance of it.
![image](https://cloud.githubusercontent.com/assets/12695977/12888904/e77c13c6-ce84-11e5-9471-2b903b2e5345.png)

Then you already have the function that return the text inherited from the class, then just use this function(which should return a bool value)/ in the test it self to validate it.